python_modules/libraries/dagster-components/dagster_components_tests/integration_tests/test_definitions_autoload.py

```diff
@@ -0,0 +1,46 @@
+import pytest
```

Comment:
most of these were just copied over from "test_definitions_component_implicit", and then the last test is net-new